TestApp/ios/Podfile.lock

```diff
@@ -1,32 +1,32 @@
 PODS:
-  - AppCenter/Analytics (1.9.0):
+  - AppCenter/Analytics (1.9.1-3):
```

Comment:
Temporary version until Native SDK released, revert that after native SDK released.